packages/node_modules/@ciscospark/internal-plugin-wdm/src/device/device.js

```diff
@@ -229,15 +229,18 @@ const Device = SparkPlugin.extend({
 
   @waitForValue('@')
   isService(service) {
-    return this._isService(this.services, service)
+    if (service === 'idbroker') {
```

Comment:
Why is `idbroker` excluded?

Reply:
@mattnorris Sorry I didn't mention this in the description. Thanks for bringing this up.

https://sqbu-github.cisco.com/gist/sgaikwad/879d8506fef720ebdf3f52b876190d8e

If you compare the old service map with the new service map carefully, you could find that there is NO idbroker in the old service map and you can see the idbroker service in the new map. 

This isService is called in function requiresCredentials in file:
spark-js-sdk/blob/master/packages/node_modules/@ciscospark/spark-core/src/interceptors/auth.js which decide whether we should add auth to the message header when we connect to idbroker (CI). So the idbroker is a special case for this function and it should not be considered as a service.